concrete/src/Application/UserInterface/Menu/Item/Item.php

```diff
@@ -17,12 +17,12 @@ public function __construct($handle, $pkgHandle = false)
 
     public function getHandle()
     {
-        return $this->handle;
+        return isset($this->handle) ? $this->handle : null;
```

Comment:
@mlocati I would just advocate for making them protected and putting a note in the backward compatibility section of the changelog.